src/search/sr_optimizers.c

```
@@ -0,0 +1,311 @@
+
+void sr_optimizer_config_init(sr_optimizer_config *cfg)
+{
+  if (!cfg) return;
+  cfg->max_evals = 0;
+  cfg->max_iters = 0;
+  cfg->seed = 0;
+}
+
+static int sr_optimizer_read_int_env(const char *name, int *out_value)
+{
+  char *end = NULL;
```

Clang (14.0.6)

[warning] variable 'end' is not initialized

(cppcoreguidelines-init-variables)